msgraph-developer-proxy-plugins/Guidance/GraphBetaSupportGuidancePlugin.cs

```diff
@@ -28,6 +28,6 @@ public class GraphBetaSupportGuidancePlugin : BaseProxyPlugin {
             _logger?.LogRequest(BuildBetaSupportMessage(request), MessageType.Warning, new LoggingContext(e.Session));
     }
 
-    private static string GetBetaSupportGuidanceUrl() => "https://learn.microsoft.com/graph/versioning-and-support#beta-version";
+    private static string GetBetaSupportGuidanceUrl() => "https://graph/proxy/guidance/beta";
```

Comment:
Shall we make it more specific, like `beta-support` or something like that in case we'll introduce more beta-related guidance, like moving beta to v1.0 endpoints when available?

Reply:
Good suggestion 👍